Specs/DateTime/German/HolidayExtractor.json

```diff
@@ -10,12 +1210,12 @@
         "Text": "Ostern",
         "Type": "date",
         "Value": {
-          "Timex": "",
+          "Timex": "XXXX",
```

Comment:
Is this the same timex returned elsewhere for non-fixed holidays?

Reply:
We return XXXX for all the non-fixed holidays that like Easter or those what depend on Easter, as easter with it "Sunday after the first full moon in spring" date is a little special.